layer/layer/canary_stack.py

```diff
@@ -49,7 +94,8 @@ def __init__(
             layers=[layer],
             memory_size=512,
             timeout=Duration.seconds(10),
-            runtime=Runtime.PYTHON_3_9,
+            runtime=Runtime.PYTHON_3_9(),
```

Comment:
```suggestion
            runtime=Runtime.PYTHON_3_9,
```

ref: https://github.com/awslabs/aws-lambda-powertools-python/blob/c31f5138b5fa2bac589c7e3b58d236abe536bb6c/tests/e2e/utils/infrastructure.py#L109